python/ray/experimental/channel.py

```diff
@@ -11,6 +11,23 @@
 logger = logging.getLogger(__name__)
 
 
+class ArgsKwargsWrapper:
+    """Warpper class for args and kwargs"""
+
+    args: List[Any]
+    kwargs: Dict[str, Any]
+
+    def __init__(self, args: List[Any], kwargs: Dict[str, Any]) -> None:
```

Comment:
Since Dict is not ordered, it'd be better to just pass in (self, args: List[Any]), and the caller can map each kwarg in the Dict to an arg index.